boards/xtensa/intel_adsp_cavs15/tools/flash.sh

```diff
@@ -15,5 +15,9 @@ elif [ -n "$3" ] && [ -n "$4" ]
     echo "Signing with key " $key
     west sign -d ${BUILD} -t rimage -p $4 -D $3 -- -k $2 --no-manifest
 fi
-echo ${FLASHER} -f ${FIRMWARE}
-${FLASHER} -f ${FIRMWARE} || /bin/true  2>&1
+
+# Make the log output can be caught by non-root permission
+ssh root@localhost chmod 666 /sys/bus/pci/devices/0000:00:0e.0/resource4
+
+echo ${FLASHER} ${FIRMWARE}
```

Comment:
Replace this line with `set -x`

Please also run `shellcheck` on this script to catch quoting and other mistakes.

Reply:
Thanks for correcting and sharing that!